SumoLogic: allow disabling url escape (#16368)
demisto/content: Packs/SumoLogic/ReleaseNotes/1_1_0.md

#### Integrations
##### SumoLogic

> - **Breaking Change:** Added the ***Escape URLs*** argument to the ***search*** command. When set to `true`, it adds a `\\` prefix to `=` chars in queried URLs.
> 
> _Instances set with previous versions of this pack do perform this escaping. To preserve this behavior, set this argument to `true`._

Reviewer: ```suggestion
- **Breaking Change:** Added the ***Escape URLs*** parameter. When set to `true`, it adds a `\\` prefix to `=` chars in queried URLs.
```
